Tasks/NuGetAuthenticateV1/main.ts

```diff
@@ -1,6 +1,10 @@
 import * as path from 'path';
 import * as tl from 'azure-pipelines-task-lib/task';
-import { installCredProviderToUserProfile, configureCredProvider } from 'azure-pipelines-tasks-artifacts-common/credentialProviderUtils'
+import * as url from 'url';
+#if WIF
+import { configureEntraCredProvider } from "azure-pipelines-tasks-artifacts-common/credentialProviderUtils";
+#endif
+import { installCredProviderToUserProfile, configureCredProvider} from 'azure-pipelines-tasks-artifacts-common/credentialProviderUtils'
 import { ProtocolType } from 'azure-pipelines-tasks-artifacts-common/protocols';
 import { getPackagingServiceConnections } from 'azure-pipelines-tasks-artifacts-common/serviceConnectionUtils'
 import { emitTelemetry } from 'azure-pipelines-tasks-artifacts-common/telemetry'
```

Comment:
Can we emit telemetry of the feature usage similar to how Emily did it in pip?

Reply:
Added a count of federated credentials we've authenticated